packages/amplify-appsync-simulator/src/server/subscription/mqtt-server/server.ts

```diff
@@ -97,7 +97,7 @@ export class MQTTServer extends EventEmitter {
 
   constructor(options: MQTTServerOptions = {}, private callback = (err: any, data?: any) => {}) {
     super();
-    this.options = extend(true, {}, DEFAULT_OPTIONS, options);
+    this.options = extend({}, DEFAULT_OPTIONS, options);
```

Comment:
May be [`_.defaultsDeep`](https://lodash.com/docs/4.17.15#defaultsDeep) a better choice

Reply:
Good choice @yuth 👍 Changed according to your suggestion.